src/crimson/os/seastore/lba_manager/btree/btree_range_pin.h

```diff
@@ -21,7 +21,7 @@ struct lba_node_meta_t {
   bool is_parent_of(const lba_node_meta_t &other) const {
     return (depth == other.depth + 1) &&
       (begin <= other.begin) &&
-      (end >= other.end);
+      (end > other.begin);
```

Comment:
so finally, we will update parent node meta to include leaf node meta, right?

Reply:
Sorry?  No, this is just fixing the is_parent_of predicate to be correct.